01-arch/07-reputation.md

```diff
@@ -2,6 +2,11 @@
 
 NeoFS reputation system is a subsystem for calculating trust in a node. It is based on a reputation model for assessing trust, which is, in turn, based on the EigenTrust algorithm designed for peer-to-peer reputation management. The algorithm ensures that there is a uniquely defined manager (parent node) for each network participant at each specified time (epoch). Based on the information received from its child node and other managers, it iteratively puts a complex general (Global) Trust of the entire network into the applicable child node.
 
+The reputation system makes it possible to determine the metric of the quality of the nodes (trust). In a situation when most of the nodes make an honest assessment of the actions of other nodes, the system allows you to calculate this metric quite accurately. This metric can be used for:
```

Comment:
The reputation system allows to introduce nodes performance quality metric (trust). 
... the system lets you calculate...

Reply:
what is better: `allows introducing` or `allows to introduce`?

Reply:
Both are ok. Use whatever you like.

Reply:
ok, fixed then